src/calendar-utils.ts

```diff
@@ -679,7 +679,7 @@ function getWeekViewHourGrid(
         )
       );
 
-      const width = 100 / columnCount;
+      const width = 90 / columnCount;
       return { ...event, left: event.left * width, width };
```

Comment:
This is technically a breaking change so can we make it an option instead that gets passed in through the outer functions and default it to 100 to preserve backwards compatibility.

Reply:
sure